src/hotspot/share/runtime/orderAccess.hpp

```diff
@@ -243,6 +243,17 @@ class OrderAccess : public AllStatic {
   static void     fence();
 
   static void     cross_modify_fence();
+
+  // Processors which are not multi-copy-atomic require a full fence
+  // to enforce a globally consistent order of Independent Reads of
+  // Independent Writes. Please use only for such patterns!
+  static void     loadload_for_iriw() {
```

Comment:
So why is this (lower case) 'iriw' when we almost always
talk about it as IRIW?